app/views/drivers/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<h2>Driver Details</h2>
```

Comment:
This collection of data should be inside of one semantic HTML element to encapsulate it